.opencode/command/commit.md

```
@@ -0,0 +1,154 @@
+예시:
+
+```markdown
+## Suggested Commits
+
+### Option 1 (recommended)
+```
+feat: Add automatic token refresh
+
+Refresh access tokens before session expiry so long-running usage
+does not unexpectedly force the user to log in again.
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)